src/components/SearchBoxMobile.js

```diff
@@ -0,0 +1,34 @@
+import SearchIcon from '@mui/icons-material/Search';
+import { Box, IconButton, InputBase } from '@mui/material';
+import React from 'react';
+
+export default function SearchBox() {
+  return (
+    <Box>
+      <InputBase
+        sx={{
+          height: '44px',
+          width: '356px',
+          borderRadius: '50px',
+          paddingLeft: '20px',
+          position: 'absolute',
+          left: '0%',
+          right: '0%',
+          top: '0%',
+          bottom: '0%',
```

Comment:
These positioning styles should not be in a reuseable component like this. This component should be a generic search bar that can be placed anywhere on any page. The parent component that implements this component will handle the placement of it.

Reply:
I have removed the positioning parameters. I kept the padding because that is used to make sure the text inside the box is not all the way to the left. I have attached the screenshot of how it looks below.

![image](https://user-images.githubusercontent.com/61099792/148148826-63734738-9ba3-4fd5-9d69-d59d2e3eda2d.png)